exercises/matrix/canonical-data.json

```diff
@@ -0,0 +1,79 @@
+{
+  "exercise": "matrix",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "extract row from one number matrix",
+      "property": "row",
+      "input": {
+        "string": "1",
+        "index": 0
+      },
+      "expected": [1]
+    },
+    {
+      "description": "can extract row",
+      "property": "row",
+      "input": {
+        "string": "1 2\n3 4",
+        "index": 1
+      },
+      "expected": [3,4]
```

Comment:
this line is missing the space after its comma (and the space is present in other lines)

Reply:
I'm not sure what you mean, I don't see a space after the comma in any other line.

Reply:
line 39. 

`"expected": [7, 8, 9]`

line 57.

`"expected": [3, 6, 9]`